src/Microsoft.ML.Data/DataLoadSave/Transpose/TransposeLoader.cs

```diff
@@ -793,7 +793,7 @@ private void Init(int col)
                 Ch.Assert(0 <= col && col < Schema.Count);
                 Ch.Assert(_colToActivesIndex[col] >= 0);
                 var type = Schema[col].Type;
-                Ch.Assert(((VectorType)type).GetValueCount() == _parent._header.RowCount);
+                Ch.Assert(((ITransposeDataView)_parent).GetSlotType(col).Size == _parent._header.RowCount);
```

Comment:
This line doesn't seem to be covered by a test, can you please add one?

https://codecov.io/gh/dotnet/machinelearning/commit/62e0b1f1be4f18b70491c4912d4028727f9992b2

Reply:
@codemzs  I added some tests, can you unblock me?